test/unit/sources/alternates/null_test.rb

```diff
@@ -6,18 +6,24 @@ class NullTest < ActiveSupport::TestCase
       alternate_should_work(
         "http://oremuhax.x0.com/yoro1603.jpg",
         Sources::Alternates::Null,
-        "http://oremuhax.x0.com/yoro1603.jpg",
-        force_https: false
+        "http://oremuhax.x0.com/yoro1603.jpg"
       )
+
+      should "not convert to https" do
```

Comment:
What exactly are you testing here? This is already covered by the alternate check above, since the link will or will not be https. Some below.

Reply:
True, yeah. I think it was different before? I forget. I can just rip these out then, that's cleaner